usagov/vote-gov-drupal
# VOTE-2880/2881: Configure and style call out box

## Jira ticket

[VOTE-2880](https://cm-jira.usa.gov/browse/VOTE-2880)
[VOTE-2881](https://cm-jira.usa.gov/browse/VOTE-2881)

## Description

Configures the display of the callout box content and styles the box.

## Deployment and testing

### Post-deploy steps

1. lando retune
2. cd into theme, npm run build

### QA/Testing instructions

1. Log into the cms on your local
2. Create a new state display content block
3. Set the following content for confirm registration
![image](https://github.com/user-attachments/assets/901cd960-9a5e-4b4f-aa2a-4e2146c07442)
4. Visit a state page and confirm the content is displaying correctly with the state name and state specific link.
5. Confirm the callout box matches the designs for desktop and mobile.

## Checklist for the Developer




## Checklist for the Peer Reviewers
